packages/frontend/src/components/staking/components/StakingAction.js

```diff
@@ -36,6 +39,10 @@ export default function StakingAction({
     currentValidators,
     nearTokenFiatValueUSD
 }) {
+    const dispatch = useDispatch();
+
+    const [checkingValidator, setCheckingValidator] = useState(false);
+    const [disableStaking, setDisableStaking] = useState(false);
```

Comment:
`validatorHasFAK` is a function, so I called it `validatorMightHaveFAK` - because it will be true if the validator has FAK or we are unable to check it.